sale_loyalty_initial_date_validity/tests/test_sale_loyalty_initial_date_validity.py

```diff
@@ -0,0 +1,116 @@
+# Copyright 2023 Tecnativa - Pilar Vargas
+# License AGPL-3.0 or later (https://www.gnu.org/licenses/agpl).
+from datetime import date, timedelta
+
+from odoo.exceptions import ValidationError
+from odoo.tests import Form, TransactionCase
+
+
+class TestSaleLoyaltyInitialDateValidity(TransactionCase):
+    @classmethod
+    def setUpClass(cls):
+        super().setUpClass()
+        loyalty_program = cls.env["loyalty.program"]
```

Comment:
If we don't check mail stuff let's disable it
```suggestion
        cls.env = cls.env(context=dict(cls.env.context, **DISABLED_MAIL_CONTEXT))
        loyalty_program = cls.env["loyalty.program"]
```

Reply:
done :)